File: PageObjectDotNet/PHPTravel/PageObjects/FeaturedTour.cs
using OpenQA.Selenium;
using OpenQA.Selenium.Support.PageObjects;
using System.Collections.Generic;
using System.Globalization;

namespace PHPTravel.PageObjects
{
    public class FeaturedTour
    {
        [FindsBy(How = How.CssSelector, Using = "div.hotel-image a div.img img")]
        private IWebElement imageMainTour;

        [FindsBy(How = How.CssSelector, Using = "div.hotel-body h3")]
        private IWebElement labelTourTitle;

        [FindsBy(How = How.CssSelector, Using = "div.hotel-body")]
        private IWebElement labelLocation;

        [FindsBy(How = How.CssSelector, Using = "span.stars i.fa-star")]
        private IList<IWebElement> imagesStars;

        [FindsBy(How = How.CssSelector, Using = "div.hotel-person span small")]
        private IWebElement labelCurrency;

        [FindsBy(How = How.CssSelector, Using = "div.hotel-person span")]
        private IWebElement labelPrice;

        [FindsBy(How = How.CssSelector, Using = "div.hotel-person ~ a.btn-block")]
        private IWebElement buttonBookNow;



        public string GetTourName()
        {
            return labelTourTitle.Text;
        }

        public int GetNumberOfStars()
        {
            return imagesStars.Count;
        }

        public string GetCurrency()
        {
            return labelCurrency.Text;
        }

        public double GetPrice()
        {
            var txt = labelPrice.Text.Split(' ')[1];
            return double.Parse(txt, NumberStyles.AllowThousands);
        }

        public TourDetails BookNow()
        {
            buttonBookNow.Click();
            return new TourDetails();
        }

    }

}

File: PageObjectJava/src/main/java/PageObjects/FeaturedTour.java
package PageObjects;

import org.openqa.selenium.WebElement;
import org.openqa.selenium.support.FindBy;

import java.util.List;

public class FeaturedTour {

    @FindBy(css = "div.hotel-image a div.img img")
    private WebElement imageMainTour;

    @FindBy(css = "div.hotel-body h3")
    private WebElement labelTourTitle;

    @FindBy(css = "div.hotel-body")
    private WebElement labelLocation;

    @FindBy(css = "span.stars i.fa-star")
    private List<WebElement> imagesStars;

    @FindBy(css = "div.hotel-person span small")
    private WebElement labelCurrency;


    @FindBy(css = "div.hotel-person span")
    private WebElement labelPrice;

    @FindBy(css = "div.hotel-person ~ a.btn-block")
    private WebElement buttonBookNow;



    public String getTourName()
    {
        return labelTourTitle.getText();
    }

    public int getNumberOfStars()
    {
        return imagesStars.size();
    }

    public String getCurrency()
    {
        return labelCurrency.getText();
    }

    public double getPrice()
    {
        String txt = labelPrice.getText().split( " ")[1];
        return Double.parseDouble(txt);
    }

    public TourDetails bookNow()
    {
        buttonBookNow.click();
        return new TourDetails();
    }

}
